### ISSUE TITLE
`exclude_paths` example fails

### ISSUE THREAD

**Issue by gregtkogut** [xlwings/jsondiff#85]
I tried to use the `exclude_paths` keyword in my code, and got a `TypeError` exception.

The README example gives the same result.

```
   1083     """
   1084     Computes the difference between two JSON structures using a specified JsonDiffer class.
   1085 
   (...)
   1091     :return: The computed diff.
   1092     """
-> 1093     return cls(**kwargs).diff(a, b, fp)

TypeError: JsonDiffer.__init__() got an unexpected keyword argument 'exclude_paths'
```

**Comment by gregtkogut** [comment]
I resolved my issue by switching to the method used by the pytest test.    So it might just be a documentation issue.

If there's concurrence I'll be happy to send a PR for the documentation.

**Comment by corytodd** [comment]
Agreed, the example is definitely wrong. I wonder if we should include a kwargs in the JsonDiffer constructor then also pass kwargs to the diff function? That would keep everything consistent and flexible, what do you think @payam54  ?
